test/NuGet.Clients.Tests/NuGet.SolutionRestoreManager.Test/VsProjectProperty.cs

```diff
@@ -15,7 +15,7 @@ public VsProjectProperty(string name, string value)
         {
             if (string.IsNullOrEmpty(name))
             {
-                throw new ArgumentException(ProjectManagement.Strings.Argument_Cannot_Be_Null_Or_Empty, nameof(name));
+                throw new ArgumentException("Argument cannot be null or empty", nameof(name));
```

Comment:
Yeah, it's the same string. I don't think we need to use localized strings in tests, let me know if there is a reason to.